react/store/createReturnRequest/components/ConfirmPaymentMethods.tsx

```diff
@@ -7,9 +7,6 @@ interface Props {
 }
 
 const messages = defineMessages({
-  bank: {
-    id: 'store/return-app.confirm-payment-methods.refund-method.p-bank',
-  },
   card: {
```

Comment:
I think we need this flag, don't we? Also, we have the same values in another component. We should use the same strategy that we used on Reason and Condition (a centralized message).